types.go

```diff
@@ -172,6 +172,8 @@ type Account interface {
 	// Expiry returns the expiry for the account in Unix Time Seconds.
 	// 0 never expires
 	Expiry() int64
+	// Encode generates a JWT string signed by the key that originally made the account
+	Encode() (string, error)
```

Comment:
For `User` we have `Creds(expiry time.Duration)`, mostly because creds is a different file, but for account we should possibly have an `Issue(time.Duration)` which would generate the JWT properly signed, then the expiration is tracked on that, with all the other settings being whatever they are.

Reply:
Otherwise looks good to me.